sass/styles/main.scss

```diff
@@ -1498,3 +1498,46 @@ cq-share-dialog {
 .cq-menu-dropdown {
 	line-height: 2.1em;
 }
+
+
+
+
+
+$spot-color: #0298d3;
+
+.cq-spot{
+	width: 6px;
+	height: 6px;
+	border-radius: 50%;
+	cursor: pointer;
+	position: absolute;
+	background: $spot-color;
+	box-shadow: 0 0 0 rgba($spot-color, 0.6);
+	animation: pulse 1.6s infinite;
+}
+
+@-webkit-keyframes pulse {
```

Comment:
This is quite a lot of duplicate code btw. Find a way to shorten this down.

This code should be in `chart.scss`. It's also weird to have both `src/app.scss` and `src/main.scss`. I'm thinking we'll need to restructure that soon.

Reply:
@binarydevteam I can handle it in this PR or later, what is your idea?